{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-0306", 
                "published": "2013-05-02T10:55:05.277-04:00", 
                "id": "CVE-2013-0306", 
                "modified": "2013-05-14T23:34:10.190-04:00", 
                "summary": "The form library in Django 1.3.x before 1.3.6, 1.4.x before 1.4.4, and 1.5 before release candidate 2 allows remote attackers to bypass intended resource limits for formsets and cause a denial of service (memory consumption) or trigger server errors via a modified max_num parameter."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/189.html", 
                "id": "CWE-189", 
                "title": "Numeric Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/a:djangoproject:django:1.3:alpha1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4:alpha"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:11.10"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.3"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:12.04:-:lts"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5:alpha"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.5:beta"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.3.3"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.3.2"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.4:beta"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:12.10"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.3.1"
            }, 
            {
                "id": "cpe:/a:djangoproject:django:1.3:beta1"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:10.04:-:lts"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.64070", 
                "id": 64070
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "https://www.djangoproject.com/weblog/2013/feb/19/security/", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "http://www.debian.org/security/2013/dsa-2634", 
                    "vendor": "DEBIAN"
                }, 
                {
                    "url": "http://ubuntu.com/usn/usn-1757-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://rhn.redhat.com/errata/RHSA-2013-0670.html", 
                    "vendor": "REDHAT"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "2.9", 
                    "access complexity": "low", 
                    "confidentiality impact": "none", 
                    "authentication": "none", 
                    "exploit": "10.0", 
                    "base": "5.0", 
                    "integrity impact": "none", 
                    "availability impact": "partial", 
                    "access vector": "network"
                }
            ], 
            "severity": "Moderate", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2013-0306"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:18138", 
                "title": "USN-1757-1 -- python-django vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:18138"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:19205", 
                "title": "DSA-2634-1 python-django - several vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:19205"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Debian Security Advisory DSA 2634-1 (python-django - several vulnerabilities)", 
                "file": "deb_2634.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "892634"
            }, 
            {
                "name": "Fedora Update for python-django FEDORA-2013-2843", 
                "file": "gb_fedora_2013_2843_python-django_fc18.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "865449"
            }, 
            {
                "name": "Fedora Update for Django FEDORA-2013-2874", 
                "file": "gb_fedora_2013_2874_Django_fc17.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "865457"
            }, 
            {
                "name": "Ubuntu Update for python-django USN-1757-1", 
                "file": "gb_ubuntu_USN_1757_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "841353"
            }
        ], 
        "Nessus": [
            {
                "name": "Debian DSA-2634-1 : python-django - several vulnerabilities", 
                "file": "debian_DSA-2634.nasl", 
                "family": "Debian Local Security Checks", 
                "id": "64898"
            }, 
            {
                "name": "Fedora 18 : python-django-1.4.5-2.fc18 (2013-2843)", 
                "file": "fedora_2013-2843.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "65232"
            }, 
            {
                "name": "Fedora 17 : Django-1.4.5-1.fc17 (2013-2874)", 
                "file": "fedora_2013-2874.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "65233"
            }, 
            {
                "name": "FreeBSD : django -- multiple vulnerabilities (21c59f5e-7cc5-11e2-9c11-080027a5ec9a)", 
                "file": "freebsd_pkg_21c59f5e7cc511e29c11080027a5ec9a.nasl", 
                "family": "FreeBSD Local Security Checks", 
                "id": "64873"
            }, 
            {
                "name": "openSUSE Security Update : python-django (openSUSE-SU-2013:1203-1)", 
                "file": "openSUSE-2013-589.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "75089"
            }, 
            {
                "name": "Ubuntu 10.04 LTS / 11.10 / 12.04 LTS / 12.10 : python-django vulnerabilities (USN-1757-1)", 
                "file": "ubuntu_USN-1757-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "65096"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [], 
        "Redhat": [
            {
                "Redhat": {
                    "oval": "", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2013-0670.html", 
                    "id": "RHSA-2013:0670", 
                    "title": ""
                }
            }
        ], 
        "Gentoo": [], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [
            {
                "url": "http://www.ubuntu.com/usn/USN-1757-1", 
                "id": "USN-1757-1"
            }
        ], 
        "Debian": [
            {
                "url": "https://security-tracker.debian.org/tracker/DSA-2634", 
                "id": "DSA-2634"
            }
        ], 
        "Vmware": []
    }
}